athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneMetadataHandlerTest.java

```diff
@@ -0,0 +1,251 @@
+/*-
```

Comment:
remove the .DS_Store file. I couldn't comment on that file directly since it is binary so commenting here as it is the next file in the list.

Reply:
ok, will delete this

Reply:
done